### PR TITLE
OE-121 OE-125 Tweaks and improvements for OE

### PR DESCRIPTION
**Pull request by halfninja**

### CODE REVIEW

**Comment by halfninja** [comment]
- Add moment TZ plugin
- Add hooks to customise CSS entrypoints
- Fix handling of dynamic chunks so they can be cached properly
- Other niceties

File: package.json [hunk 1]
```diff
@@ -22,21 +22,22 @@
...
     "moment-locales-webpack-plugin": "1.1.0",
+    "moment-timezone-data-webpack-plugin": "^1.1.0",
     "optimize-css-assets-webpack-plugin": "5.0.1",
     "postcss-loader": "3.0.0",
     "postcss-safe-parser": "4.0.1",
     "purify-css": "1.2.5",
     "purifycss-webpack": "0.7.0",
     "terser-webpack-plugin": "1.2.1",
-    "webpack": "4.28.3",
+    "webpack": "^4.42.1",
```
Comment (flamingelk) [inline package.json:32]: Is this deliberately set to auto-update?

File: package.json [hunk 2]
```diff
@@ -22,21 +22,22 @@
     "less-loader": "4.1.0",
     "mini-css-extract-plugin": "0.4.5",
     "moment-locales-webpack-plugin": "1.1.0",
+    "moment-timezone-data-webpack-plugin": "^1.1.0",
```
Comment (matmannion) [inline package.json:25]:
```suggestion
    "moment-timezone-data-webpack-plugin": "1.1.0",
```

File: package.json (hunk 1 above)
Comment (matmannion) [inline package.json:32]:
```suggestion
    "webpack": "4.42.1",
```

Comment (halfninja) [reply package.json:32]: Yes, it doesn't matter if a newer 4.x version is used as long as it gets a new enough version to have the logging API that's used

**Pull request merged**